src/Tool/Program.cs

```diff
@@ -35,6 +36,28 @@ public static bool TelemetryOptOut
             => true;
 #endif
 
+        // Creates dictionary of kernelspec file names to the embedded resource path
+        // of all embedded resources found in `Kernel.csproj`.
```

Comment:
It's best for documentation like this to use triple-slash notation so that it gets picked up by IntelliSense, doc generation, etc. Like:
```
/// <summary>
/// Text here.
/// </summary>